notifier/core/notifierapp.py

```diff
@@ -1,25 +1,41 @@
 import os
 from celery import Celery
 from .config import Config
-from .private.singleton import Singleton
 
 
-@Singleton
 class NotifierApp(Celery):
     def __init__(self):
-        self.__config = Config.instance()
-        super().__init__(self.__config.key("app_name"))
-        self.conf.update(broker_url=self.__config.key("broker_url"),
+        config = Config()
+        super().__init__(config.key("app_name"))
+        self.conf.update(broker_url=config.key("broker_url"),
                          result_backend="rpc://")
 
+        self.__namespace = "Notifier"
+        self.__queue = config.key("queue_name")
+        self.__plugin_path = config.key("plugin_path")
+        self.__log_from = config.key("log_from")
+        self.__log_file = config.key("log_file")
+        self.__pid_file = config.key("pid_file")
+        self.__hostname = config.key("hostname")
+        self.__workers = config.key("num_workers")
```

Comment:
Nice way to get the config file params! 👍
I wonder, should we add some default values just in case the value is not set in the config file? I mean if num_workers is not set in the config, it should be set as 4 by default..., and the same with the others, thus, we would not need to create any config file in order to run the notifier app

Reply:
Fix de4c258